app/scripts/views/mixins/signin-mixin.js

```diff
@@ -103,6 +114,12 @@ define(function (require, exports, module) {
             return this.navigate('signin_bounced', { email: account.get('email') });
           }
 
+          if (AuthErrors.is(err, 'TOTP_REQUIRED')) {
+            err.forceMessage = t('This request requires two step authentication enabled on your account. ' +
+              '<a href=\'' + TOTP_SUPPORT_URL + '\'>More Information</a>');
```

Comment:
You could just use an ES6 string here:

```js
`<a href="${TOTP_SUPPORT_URL}">More Information</a>`
```

Can you also wrap TOTP_SUPPORT_URL in a call to `XSS.href` as a defense in depth measure in case we ever make some change to the above URL, or wrap it above [similar to how the password strength balloon does it](https://github.com/mozilla/fxa-content-server/blob/84e6ea9104ec381250f9c88d07988a8e805076d3/app/scripts/views/password_strength/password_strength_balloon.js#L28)?

```js
`<a href="${XSS.href(TOTP_SUPPORT_URL)}">More Information</a>`
```